### ISSUE TITLE
I broke my workshops after I installed the "n" module

### ISSUE THREAD

**Issue by horkage** [nodeschool/discussions#1047]
I don't know if this is the appropriate place to ask this...apologies if it is not..

I installed node a little while back - version 0.10.36 on mac os x.
A couple weeks later, I installed some nodeschool workshops and started working my way through them.
Everything was perfect up until the stream-adventure http server challenge, which was blowing up on me because I was using an older version of node (0.10.36).

So, I installed the 'n' module and used it to install node versions 0.12.2 and 0.10.35.  Now my nodeshool workshops don't work anymore (with either version of node) - they either crash or hang indefinitely which, to me, indicates that I must've really borked something.

Does anyone have any ideas?  All my nodeschool workshops and the 'n' module were installed globally via sudo.  I installed the different versions of node via 'n' with sudo as well.

Thanks for any help!

**Comment by martinheidegger** [comment]
'n' is a command line tool that twists and tweaks your PATH variable and other things. That is not a cool thing to do :)

I am not sure what exactly your problem is (from the given information its hard to dissect it) perhaps you need to reinstall the workshoppers? `npm r learnyounode -g` and `npm i learnyounode -g`

In any case: It would help to see the error mesages with which the workshoppers "crash". Can you copy&paste them?

**Comment by horkage** [comment]
Thanks for you feedback - I tried as you recommended (remove/reinstal/etc) but it didn't seem to work.  The errors weren't exactly errors per se, but issues like io hanging and in particular, the baby steps challenge - you write a program that expects arguments on process.argv and sums them - but when I was running verify/run, I wasn't receiving those arguments from the test program like I was before.

At any rate, I found and cleared out that ~/.config/learnyounode directory, (which looks like it shouldn't have any impact like I'm seeing), and now my baby steps challenge is working again - (argv has stuff now).  

So, I'm going to try and run through them all and see if clearing that directory did the trick.

Thanks again
